### DISCUSSION TITLE
Galaxy brain

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#143] [Q&A] [answered]
Help me here

**Comment by montasim** [comment]
Get one

**Comment by fcasamento** [comment, marked as answer]
@montasim, I'm over here! By the way, which devops tracks are you studying besides and of course github?

**Reply by montasim** [reply]
Hello, @fcasamento. Currently, I am focusing on MERN Stack and working as a Frontend Developer.

**Reply by fcasamento** [reply]
That's great, man

**Reply by montasim** [reply]
Hey, @fcasamento, I have sent you a connection request on LinkedIn. I will be very happy to connect with you.

**Reply by fcasamento** [reply]
@montasim, Do you have any intentions of studying devops, or some of the technologies involved? Git and github, not worth it, lol.

**Reply by montasim** [reply]
Hey @fcasamento, I have a fascination with DevOps. So, why not give it a try? Can you please share more details about it? My email is [montasimmamun@gmail.com](mailto:montasimmamun@gmail.com)

**Reply by fcasamento** [reply]
@montasim Sure, friend. I'll share the basics first to get you started and let's talk there.

**Reply by montasim** [reply]
Thank you very much @fcasamento. Your help is much appreciated.
